docs/en/core-services/Caching.md

```diff
@@ -144,12 +144,18 @@ The user can also specify the tier that the data will be written to via
 
 If the data is already in Alluxio, the client will simply read the block from where it is already stored.
 If Alluxio is configured with multiple tiers, the block may not be necessarily read from the top tier,
-since it could have been moved to a lower tier transparently.
+since it may have been loaded into a lower tier, or moved to a lower tier transparently. 
+This logic applies to both `ReadType.CACHE` and `ReadType.CACHE_PROMOTE`.
 
-Reading data with the `ReadType.CACHE_PROMOTE` will attempt to first transfer the block to the
+The difference is, reading data with `ReadType.CACHE_PROMOTE` will attempt to first transfer the block to the
 top tier before it is read from the worker. This can also be used as a data management strategy by
 explicitly moving hot data to higher tiers.
 
+For `ReadType.CACHE`, Alluxio will cache the block into the highest tier that has the available space.
+So you will read the cache block with disk speed if the block is currently on a disk(SSD/HDD).
+
+> Note: In 2.3, Alluxio default ReadType is changed from CACHE_PROMOTE to CACHE.
```

Comment:
@gpang  If `CACHE_PROMOTE` moves the block up to MEM tier so only the first read on the existing block is disk space and subsequent ones are supposed to be memory speed, does that imply `CACHE_PROMOTE` will be faster in some scenarios? I don't find where we talk about that.

Reply:
Can you explain why it was changed to CACHE? It was because the async alignment tasks will readjust blocks according to the annotation policy, so we don't require the synchronous move to top tier, since that can cause unnecessary delays.

Reply:
I just added more explanation. Does it make more sense now?